pulsar-client-cpp/lib/Schema.cc

```diff
@@ -90,6 +117,51 @@ SchemaInfo::SchemaInfo(SchemaType schemaType, const std::string &name, const std
                        const StringMap &properties)
     : impl_(std::make_shared<SchemaInfoImpl>(schemaType, name, schema, properties)) {}
 
+SchemaInfo::SchemaInfo(const SchemaInfo &keySchema, const SchemaInfo &valueSchema,
+                       const KeyValueEncodingType &keyValueEncodingType) {
+    auto checkType = [](const SchemaInfo &schemaInfo) {
+        if (schemaInfo.getSchemaType() != JSON && schemaInfo.getSchemaType() != AVRO) {
+            throw std::invalid_argument("Key and value schema just support JSON or AVRO.");
```

Comment:
It's better to add this information to the PR description.

Reply:
Ok, This behavior remains consistent with the java implementation.